docs/guide/docs/developers/pipeline.md

```diff
@@ -0,0 +1,168 @@
+---
+id: pipelines
+title: Bot Pipeline
+---
+
+## Overview
+
+Bot pipelines are built on top of the multi-bot capability of Botpress. They allow teams to work on bots with multiple **stages** the same way others would work on software products. Different versions of a bot can run and co-exist on different stages. In software development, each team has its own way of working with pipelines and reacting to events along the pipeline, Botpress pipelines let you do the same.
+
+Although a typical pipeline would have 3 stages (e.g: _Development_, _Staging_ and _Production_) where a bot goes through all of them Botpress let you define your own stages on a single spreaded across multiple Botpress deployement.
+
```

Comment:
Not sure to fully understand the last sentence

Reply:
right couple of words are missing ><